flink-runtime-web/web-dashboard/src/app/pages/job/job-local.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from '@angular/core';
+import { combineLatest, Observable, ReplaySubject } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
+
+import { JobDetailCorrect, NodesItemCorrect } from 'interfaces';
+
+type Nullable<T> = T | null;
+
+interface JobWithVertex {
```

Comment:
I think we should export types that are referenced in public APIs, otherwise the consuming code cannot reference the type.